app/controllers/exercises_controller.rb

```diff
@@ -1,21 +1,50 @@
 class ExercisesController < ApplicationController
+  before_action :set_exercise, only: [:edit, :update, :destroy]
   def index
-    @exercises = Exercise.all
+    @exercises = @restaurants = policy_scope(Exercise)
   end
 
   def new
-    raise
+    @exercise = current_user.exercises.new
+    authorize @exercise
   end
 
   def create
-    raise
+    @exercise = current_user.exercises.new(exercise_params)
+    authorize @exercise
+
+    if @exercise.save
+      redirect_to exercises_path
+    else
+      render :new
+    end
   end
 
   def edit
-    raise
   end
 
   def update
+    @exercise.update(exercise_params)
+
+    if @exercise.save
+      redirect_to exercises_path
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @exercise.destroy
+    redirect_to exercises_path
+  end
+
+  def exercise_params
     raise
```

Comment:
Can you make sure to remove your `raise`. There is no reason to merge a `raise` to master. Empty actions can just be `def action; end`

Reply:
If it's not ready then it shouldn't be touched because this creates issues for other people. This is something I could very easily do by myself.

I have to keep removing raises because it crashes. If you need to remind yourself of something, put comments. I don't think merging debugging code on main is good development practice.